### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by cuauh-cabrera**
Below are listed all the features included in this branch:

- [ ] Add headline for mobile devices.
- [ ] Add linters.
- [ ] Add README file.

### CODE REVIEW

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @cuauh-cabrera, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your portfolio looks good, welldone 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Cuau Cabrera - Software Developer</title>
+</head>
+<body>
+    <header class="header">
+        <img src="./assets/Toolbar - Dark - Compact.png" class="header__toolbar" alt="">
+    </header>
```
[the remark this answers is not in the document]
Comment (cuauh-cabrera) [reply index.html]: - [ ] Add <nav> tag
- [ ] Change the menu button position.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                <a href="https://linkedin.com/in/cuaucabrera"><img src="./assets/Linkedin icon.png"
+                        alt="This is a Linkedin icon. Click on it to visit my Linkedin profile"
+                        class="headline__icons--linkedin"></a>
+                <a href="https://github.com/cuauh-cabrera"><img src="./assets/github.png"
+                        alt="This is a github icon. Click on it to visit my github profile"></a>
+            </div>
+            <div class="headline__action">
+                <img src="./assets/Action.png" alt="Click on this icon to move to the next section">
+            </div>
+        </div>
```
[the remark this answers is not in the document]
Comment (cuauh-cabrera) [reply index.html]: - [ ] Include <section> tag to enclose the headline.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            <div class="headline__icons">
+                <a href="https://twitter.com/cuau_cabrera"><img src="./assets/twitter.png"
+                        alt="This is a Twitter icon. Click on it to visit my Twitter profile"
+                        class="headline__icons--twitter"></a>
+                <a href="https://linkedin.com/in/cuaucabrera"><img src="./assets/Linkedin icon.png"
+                        alt="This is a Linkedin icon. Click on it to visit my Linkedin profile"
+                        class="headline__icons--linkedin"></a>
+                <a href="https://github.com/cuauh-cabrera"><img src="./assets/github.png"
+                        alt="This is a github icon. Click on it to visit my github profile"></a>
+            </div>
```
Comment (mukhammadrizooff) [inline index.html:36]: - [OPTIONAL] Good job so far! The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag) 👍🏼

**Review by mukhammadrizooff — APPROVED** [review]
Hi @cuauh-cabrera, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**